go/worker/common/config.go

```diff
@@ -75,32 +32,19 @@ func NewConfig() (*Config, error) {
 	}
 
 	cfg := Config{
-		ClientPort:      uint16(viper.GetInt(CfgClientPort)),
 		SentryAddresses: sentryAddresses,
 		TxPool: txpool.Config{
```

Comment:
Should we just use `txpool.Config` directly in the config (could move it in a subpackage if needed)? It doesn't make much sense to have this copying here now that we are using config structures anyway?

Reply:
Yeah, that makes sense.

Reply:
I've changed txpool to use the structure defined in the config.